bufferoverride/gui/bufferoverrideview.cpp

```diff
@@ -231,12 +232,12 @@ void BufferOverrideView::draw(VSTGUI::CDrawContext *ctx) {
   }
 
   // legend label
-  const auto legend_label = [](auto window_sec) {
+  const auto legend_label = [window_sec = window_sec] {
```

Comment:
I think just [window_sec] will do the same thing?
(Or just use ?: with no lambda...)

Reply:
The former isn't the same annoying due to `window_sec` being part of a structured binding, not eligible for simple copy capture. Using a ternary expression is certainly reasonable though (and warranted; I am guessing during development I had a more complex lambda at some point? but now it seems pointless).

Reply:
Oh yeah, the structured binding thing has bit me several times. Makes sense.